internal/app/upstream/client.go

```diff
@@ -289,3 +291,19 @@ func shutDown(ctx context.Context, conn *grpc.ClientConn) {
 func (e *UnsupportedResourceError) Error() string {
 	return fmt.Sprintf("Unsupported resource typeUrl: %s", e.TypeURL)
 }
+
+func updateConnectivityMetric(ctx context.Context, conn *grpc.ClientConn, scope tally.Scope) {
+	connectedStat := "connected"
```

Comment:
can you unify this in https://github.com/envoyproxy/xds-relay/blob/master/internal/app/metrics/metrics.go?